frontend/src/component/menu/routes.ts

```diff
@@ -549,4 +532,7 @@ export const getCondensedRoutes = (routes: IRoute[]): INavigationMenuItem[] => {
     });
 };
 
-export const getRoutes = computeRoutes();
+export const baseRoutes = routes.filter((route) => !route.hidden);
+export const getNavRoutes = () => {
+    return getCondensedRoutes(baseRoutes.filter((route) => route.menu.main));
```

Comment:
Might as well export them condensed out of the box - there's only one reference to this and that also imported the getCondensed and applied it to the returned collection